galaxykit/client.py

```diff
@@ -13,6 +13,9 @@
 from . import containerutils
 from . import groups
 from . import users
+from . import namespaces
```

Comment:
```
galaxykit -s 'http://localhost:8002/api/galaxy/' -u admin -p admin collection list
Traceback (most recent call last):
  File "/home/runner/.local/bin/galaxykit", line 5, in <module>
    from galaxykit.command import main
  File "/home/runner/.local/lib/python3.8/site-packages/galaxykit/__init__.py", line 2, in <module>
    from .client import GalaxyClient
  File "/home/runner/.local/lib/python3.8/site-packages/galaxykit/client.py", line 17, in <module>
    from . import collections
  File "/home/runner/.local/lib/python3.8/site-packages/galaxykit/collections.py", line [13](https://github.com/ansible/ansible-hub-ui/runs/7508014473?check_suite_focus=true#step:17:14), in <module>
    from .client import GalaxyClientError
ImportError: cannot import name 'GalaxyClientError' from partially initialized module 'galaxykit.client' (most likely due to a circular import) (/home/runner/.local/lib/python3.8/site-packages/galaxykit/client.py)
```

looks like collections is importing client, so client can't import collections ... we probably need to move some exports to a separate file?